src/utilities/__tests__/buildClientSchema-test.js

```diff
@@ -530,6 +530,18 @@ describe('Type System: build schema from introspection', () => {
     expect(cycleIntrospection(sdl)).to.equal(sdl);
   });
 
+  it('builds a schema with specified by', () => {
+    const sdl = dedent`
+      type Query {
+        someUUID: UUID
+      }
+
+      scalar UUID @specified(by: "https://tools.ietf.org/html/rfc4122")
```

Comment:
If at some point we decide to add `UUID` scalar into this library it would be a confusing example.
It's better to use some abstract scalar like `Foo` and `https://example.com/foo_spec`.